shell/platform/darwin/ios/ios_external_texture_gl.mm

```diff
@@ -22,6 +22,23 @@
 
 IOSExternalTextureGL::~IOSExternalTextureGL() = default;
 
+void IOSExternalTextureGL::CreateTextureFromPixelBuffer() {
+  if (buffer_ref_ == nullptr) {
+    return;
+  }
+  CVOpenGLESTextureRef texture;
+  CVReturn err = CVOpenGLESTextureCacheCreateTextureFromImage(
+      kCFAllocatorDefault, cache_ref_, buffer_ref_, nullptr, GL_TEXTURE_2D, GL_RGBA,
```

Comment:
I don't understand where this cache is being recreated. I thought it was collected in IOSExternalTextureGL::OnGrContextDestroyed.

Reply:
Ah, I did not recreate the texture cache. It would still get created on the next frame, but not ideal. Will fix it.